### PR TITLE
feat: add flag to indicate if connector is external

### PR DESCRIPTION
**Pull request by arthurgeron**
# Release notes

In this release, we:

- Added flag to indicate if connector is external

# Summary

Add external flag to differentiate between native and external connectors.

# Breaking Changes

N/A

# Checklist

- [x] All **changes** are **covered** by **tests** (or not applicable)
- [x] All **changes** are **documented** (or not applicable)
- [x] I **reviewed** the **entire PR** myself (preferably, on GH UI)
- [x] I **described** all **Breaking Changes** (or there's none)

### CODE REVIEW

File: packages/account/src/connectors/fuel-connector.ts [hunk 1]
```diff
@@ -90,6 +90,7 @@ export abstract class FuelConnector extends EventEmitter implements Connector {
   metadata: ConnectorMetadata = {} as ConnectorMetadata;
   connected: boolean = false;
   installed: boolean = false;
+  external: boolean = true;
```
Comment (petertonysmith94) [inline packages/account/src/connectors/fuel-connector.ts:93]: Hey @arthurgeron,

Don't suppose we could get a little bit more information on what determines a connector to be external or internal?

Comment (arthurgeron) [reply packages/account/src/connectors/fuel-connector.ts:93]: External connectors are third-parties that do not support the Fuel Network.
Non-Native: Wallet Connect, Solana Connector
Native: Fuelet, Burner, Fuel

Comment (arthurgeron) [reply packages/account/src/connectors/fuel-connector.ts:93]: I've added the property to the docs

**Comment by arthurgeron** [comment]
@petertonysmith94 
Could we please get `0.94.7` released after this is merged?

**Review by Torres-ssf — APPROVED** [review]

**Review by arboleya — APPROVED** [review]

**Pull request merged**